browser-test/src/support/index.ts

```diff
@@ -498,15 +499,21 @@ export type LocalstackSesEmail = {
   Subject: string
 }
 
+/**
+ * Queries the emails that have been sent for a given recipient. This method requires that tests
+ * run in an environment that uses localstack sinc it captures the emails sent using SES and makes
+ * them available at a well-known endpoint). An error is thrown when the method is called from an
+ * environment that does not use localstack.
```

Comment:
Add that `supportsEmailInspection` can be used to determine that.

Similar to a map.get() method saying you should do map.has() first etc

Reply:
Done.